<?xml version="1.0"?>
<launch>
    <arg name="hz" default="1" />

    <node pkg="object_identifier" type="debug_object_map_publisher" name="debug_object_map_publisher" output="screen">
        <param name="HZ" value="$(arg hz)" />

        <rosparam command="load" file="$(find object_identifier)/config/params/debug_object_map.yaml" />
    </node>
</launch>
